lib/kubeclient.rb

```diff
@@ -195,7 +193,7 @@ def ssl_options(client_cert: nil, client_key: nil, ca_file: nil,
     end
 
     def all_entities
-      ENTITY_TYPES.each_with_object({}) do |entity_type, result_hash|
+      ENTITY_TYPES.each_with_object({}) do |(_, entity_type), result_hash|
```

Comment:
`ENTITY_TYPES` is a list of lists.  `(_, entity_type)` deconstructs the list.  `_` is the class we made above, and `entity_type` is a string.